### ISSUE TITLE
How to clear drawn things ?

### ISSUE THREAD

**Issue by AjayShivanagol** [lovasoa/whitebophir#259]
How do we clear the drawn things on the screen.... when we close the browser OR WBO tab?
is there any configuration for it ?

**Comment by lovasoa** [comment]
There are two separate behaviors here:

1. Manually clearing the current board is now supported by the Clear tool, which is gated by the `clearBoard` capability.
2. Automatically clearing/deleting a board when a browser tab closes is not implemented and should be tracked as board lifecycle behavior.

The automatic cleanup part is already tracked more clearly in #199 ("Delete Board as the last user leaves"). That issue is the right place to define the desired lifecycle semantics and avoid accidental data loss on temporary disconnects or reloads.

Closing this as answered/duplicate of #199 for the automatic-on-close behavior.
